lib/zold/version_file.rb

```diff
@@ -9,11 +9,26 @@ def initialize(path, log: Log::Verbose.new)
       @log = log
     end
 
-    # @todo #285:30min Replace this stub with functionality.
-    #  We need to run the script (`yield`) if the version of
-    #  the script is between the saved version and the current one.
     def apply(version)
-      @log.info("Version #{version} doesn't need to be applied.")
+      return unless Gem::Version.new(version) > current_data_version
+
+      yield version
+    end
+
+    private
+
+    def current_data_version
+      return Gem::Version.new('0') unless File.exist?(version_file)
+
```

Comment:
@ledestin again here: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html An empty line inside method body is an obvious sign of a problem in the method design. Never use them.